src/components/common/CommunityCounts.tsx

```diff
@@ -0,0 +1,50 @@
+import { CommunityAggregates } from "lemmy-js-client";
+import React from "react";
+import { HStack, useTheme, Text, VStack } from "native-base";
+import Chip from "./Chip";
+import { shortenNumber } from "../../helpers/NumberHelper";
+
+interface IProps {
+  counts: CommunityAggregates;
+}
+
+function CommunityCounts({ counts }: IProps) {
+  const theme = useTheme();
+
+  const subscribers = shortenNumber(counts.subscribers);
+  const posts = shortenNumber(counts.posts);
+  const comments = shortenNumber(counts.comments);
+  const usersActiveMonth = shortenNumber(counts.users_active_month);
+  const usersActiveWeek = shortenNumber(counts.users_active_week);
+  const usersActiveDay = shortenNumber(counts.users_active_day);
```

Comment:
@bohn002 we already had this util (but I realized it was broken, so fixed it). Also you named the function (PrettyNumbers) with PascalCase, but non React component functions should be camelCase.

React components - PascalCase
Everything else - camelCase